packages/terra-base/tests/nightwatch/base-spec.js

```diff
@@ -9,6 +9,17 @@ module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous
     browser.expect.element('div').text.to.contain('en').before(waitInms);
   },
 
+  // new
```

Comment:
this comment could be a bit more descriptive

Reply:
Removed in this commit: https://github.com/cerner/terra-core/pull/1012/commits/de92c42b10d391c997e2d2edd3ee4b606e679213